tests/test_pyproject_pep517.py

```diff
@@ -189,3 +203,31 @@ def test_pep517_wheel(tmp_path, monkeypatch, virtualenv):
         capture=True,
     )
     assert add.strip() == "3"
+
+
+@pytest.mark.skip(reason="Doesn't work yet")
```

Comment:
@agoose77 this didn't pass when I ran it locally, so I'm thinking it's not automatically handled?

Reply:
How about changing this to XFail? Isn't it expected to work as written later on?